test/evergreen.yml

```diff
@@ -1784,6 +1814,48 @@ tasks:
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
 
+  - name: recovery-stress-test
+    #set a 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict --enable-diagnostic --with-builtins=lz4,snappy,zlib
+      - func: "recovery stress test script"
+        vars:
+          # Repeat this script enough times to make this task run for 24 hours
+          # At the time of writing this script, one cal to underlying scripts takes about 8 mins to finish.
+          times: 180
+  
+  - name: recovery-stress-test-without-lz4
+    #set a 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict --enable-diagnostic --with-builtins=snappy,zlib
+      - func: "recovery stress test script"
+        vars:
+          # Repeat this script enough times to make this task run for 24 hours
+          # At the time of writing this script, one cal to underlying scripts takes about 8 mins to finish.
+          times: 180
+
+  - name: split-stress-test
```

Comment:
This is from merging. Not my changes.

Reply:
It's weird the merged change shows up as part of the PR. Did you use merge or rebase?

Reply:
I used merge but somehow this weird merge issues are showing up. I will probably revert to pre-merge state and try to redo the merge.

Reply:
Will leave to you to handle it